clang/test/CodeGenSYCL/unique-stable-name-placeholder-crash.cpp

```diff
@@ -0,0 +1,29 @@
+// RUN: %clang_cc1 -triple spir64-unknown-unknown-sycldevice -std=c++17 -sycl-std=2020 -fsycl -fsycl-is-device -disable-llvm-passes -emit-llvm %s -o - | FileCheck %s
+
+template <typename KernelName, typename KernelType>
+[[clang::sycl_kernel]] void kernel_single_task(const KernelType &kernelFunc) {
+  kernelFunc();
+}
```

Comment:
Could you convert this instead to using the mock header?

Reply:
... maybe?  Do you have an example?